packages/functional-tests/tests/subscription-tests/support.spec.ts

```diff
@@ -57,42 +69,72 @@ test.describe('severity-2 #smoke', () => {
       );
       test.slow();
 
-      await signInAccount(target, page, login, testAccountTracker);
+      const credentials = await signInAccount(
+        target,
+        page,
+        settings,
+        signinReact,
+        testAccountTracker
+      );
 
       await relier.goto();
       await relier.clickSubscribe();
-      await subscribe.setConfirmPaymentCheckbox();
-      await subscribe.setFullName();
-      await subscribe.setCreditCardInfo();
-      await subscribe.clickPayNow();
-      await subscribe.submit();
-
-      //Login to FxA account
-      await login.goto();
-      await login.clickSignIn();
-      const subscriptionPage = await settings.clickPaidSubscriptions();
-      subscriptionManagement.page = subscriptionPage;
-      await subscriptionManagement.fillSupportForm();
-      await subscriptionManagement.cancelSupportForm();
+
+      await expect(subscribe.setupSubscriptionFormHeading).toBeVisible();
+
+      await subscribe.confirmPaymentCheckbox.check();
+      await subscribe.paymentInformation.fillOutCreditCardInfo(VALID_VISA);
+      await subscribe.paymentInformation.clickPayNow();
+
+      await expect(subscribe.subscriptionConfirmationHeading).toBeVisible();
+
+      //Signin to FxA account
+      await signinReact.goto();
+
+      await expect(signinReact.cachedSigninHeading).toBeVisible();
+      await expect(page.getByText(credentials.email)).toBeVisible();
+
+      await signinReact.signInButton.click();
+      const newPage = await settings.clickPaidSubscriptions();
+      const subscriptionManagement = new SubscriptionManagementPage(
+        newPage,
+        target
+      );
+      const subscriptionSupport = new SubscriptionSupportPage(newPage, target);
+
+      await expect(subscriptionManagement.subscriptiontHeading).toBeVisible();
+
+      await subscriptionManagement.contactsupportButton.click();
+
+      await subscriptionSupport.fillOutSupportForm(
+        target.subscriptionConfig.name,
+        Topic.PAYMENT_AND_BILLING,
+        App.DESKTOP,
+        'Test Support',
+        'Testing Support Form'
+      );
+      await subscriptionSupport.cancelButton.click();
 
       //Verify it redirects back to the subscription management page
-      expect(await subscriptionManagement.subscriptiontHeader()).toBe(true);
+      await expect(subscriptionManagement.subscriptiontHeading).toBeVisible();
     });
   });
 });
 
 async function signInAccount(
   target: BaseTarget,
   page: Page,
-  login: LoginPage,
+  settings: SettingsPage,
+  signinReact: SigninReactPage,
   testAccountTracker: TestAccountTracker
 ): Promise<Credentials> {
   const credentials = await testAccountTracker.signUp();
   await page.goto(target.contentServerUrl);
-  await login.fillOutEmailFirstSignIn(credentials.email, credentials.password);
+  await signinReact.fillOutEmailFirstForm(credentials.email);
+  await signinReact.fillOutPasswordForm(credentials.password);
```

Comment:
Something I've noticed with the tests is the the `filloutXForm` type helpers almost always assert a path change after submission. It might be helpful to update these helpers to take an optional path url to assert. (Not scope for PR though)

Reply:
That could be very cool!